Added changes you may have missed to changelog

fmpreferences opened the pull request:
title

civ6bbg commented:
please send me a message in discord to double check these
my username: calcalcifer

fmpreferences commented:
6.0:
Specialist yield line 118 and Oil on plains line 69 here
https://github.com/CivilizationVIBetterBalancedGame/BetterBalancedGame/blob/main/sql/Base/base.sql

Oil on floodplains line 215
https://github.com/CivilizationVIBetterBalancedGame/BetterBalancedGame/blob/main/sql/XP2/xp2__gathering_storm.sql

6.1 

https://docs.google.com/,document/d/e/2PACX-1vSBJHFld4Z3xkcuP1i1cxjNW8TpxB6FLOtwRcUy_Nh2WdmEPyNcSgVFfWWRFNehAb6J7KJBu_puKUVa/pub

Under congress header:

Trade Policy: *BUG FIX* Vote A - Now should correctly give +4 gold to players sending trade routes to the target and the target gets +1 trade route. Previously this was the text, but firaxis had it coded that the target got the +4 gold as well as the trade route and other players received nothing.

6.2:
Final patchnotes are incomplete, so I refered to the beta updates. The changes I'm adding to the changelog highlighted in bold (the whole changelog is to confirm it was not reverted later)
<details>
<summary>FULL beta patchnotes</summary>
Civilization :
- 🔻 Brazil : Minas melee combat strength reduced to 60 (from 70, -10)
- 🔻 Byzantium, Theodora : Culture delayed to Political
- 🔻 Hungary : Huszar combat strength reduced to 63 (from 65, -2)
- 🔻 India, Chandragupta : Holy Site building do not give combat strength and general point anymore.
- 🔻 Macedon :
  - Hetaroi production cost to 55 (from 50) 
  - Science on production and kill reduced to 20% (from 25%)
-  ♻️  Nubia :
  - Nubia gets +20% production on districts (doubled when a Pyramid is in the city)
  - Pyramid can be placed on any flat tiles, limited to one per city and non adjacent to another pyramid
  - Nubian district gets standard adjacency from nubian pyramid
  - Base Nubian Pyramid is +2 food, +2 faith
  - Encampment and Aerodrome give +1 production to Pyramid, Diplo Quarter and Gov Plaza give +1 culture/2 gold, city center +1 food (from +2) + old bonuses are still here but only +1
- 🔻 Russia : remove +1 base faith on capital.
- 🔻 Scythia : combat bonus reduced to +3 base (scaling to +5 at Stirrups)
- ⬆️  Sejong :
  - Builder prod % to 30 (from 15)
  - Culture on new tech era to 2x science (from 1)
- 🔻 Sumer : remove combat bonus
- ⬆️ Vietnam : Gets +1mov and 2 combat str when on a feature everywhere on the map (not doubled in own territory). Revert Voi Chen bonus added previous patch
Governor :
- ♻️ Moksha RI : +2 prophet points and faith for cities with HS within 6 tiles of this city (remove previous bonus).
- ♻️ Moksha RII : Receive science and culture equal to 10% of the city faith output (remove previous bonus).
- 🔻Amani RII : Remove base favor/influence

Wonders :
- ⬆️ Országház : 
  - +50% influence point/turn
  - 2 emissaries
- ⬆️ Colossus : Foreign Trade Routes to this city provide +2 Gold to both cities.

Other :
- ♻️ Reeds & Marshes : now work on floodplains only for improved tiles and city center
**- ♻️ Congress : combat strength resolution down to -+3 (from -+5)**
- [NEW] Melee ancient era unit (Warrior and UUs, Scout and UU, Galley and UU, WarCart and Sabum) get -5 against city center until a classical technology is reached
**- ♻️ Pillage value has been slightly reduced on districts
- ♻️ Pillage value has been reduced on improvements**

Green Cards :
- ⬆️ Praetorium : +10% production on conquered cities with a governor (keep old bonus)
- ♻️ Colonial Offices : +15% faster growth, +10% Production and 3 Loyalty per turn for cities 8 tiles away from your capital. [Obsolete at Colonialism]
- ♻️ Colonial Taxes : +15% faster growth, +25% Gold, +10% Production and 3 Loyalty per turn for cities 8 tiles away from your capital.
- [NEW] Sovereign Trade : open Borders with all city state (obsolete at Ideology. Ideology gives open borders with all city states), unlocked at Civil Service.
- ♻️ Gunboat Diplomacy : +4 points Influence points per turn toward earning Envoys and +1 Gold from each of your Envoys at city-states. (removed the open borders with city states part but merge the card with Merchant Confederation). Merchant confederation is now obsolete at Ideology
- ♻️ Raj : +1 Prod, Food and +2 Science, Culture, Faith and Gold for Trade Route to city-states. (old bonus removed)
Hello <@&1009714827721199646> 6.2b is out :
Civilizations :
- Egypt :
  - Production bonus for district next to river reduced to 15% (from 20)
- Egypt, Cleopatra :
  - Gold for traderoute reduced to 4
  - Food for sending traderoute to Cleopatra reduced to 1
  - 15% production bonus for commercial hubs and Theater when the city is settled on a river
- Egypt, Ptolemaic :
  - Old bonus removed
  - 15% production bonus for encampment and campuses when the city is settled on a river
  - Military units ignores river debuffs (combat and movement) and gets +3 combat strength on floodplains
- Egypt, Ramses :
  - % culture from buildings and wonder untouched
  - Holy Site on a river get +1 production
  - 15% production bonus for HS and IZ when the city is settled on a river

Other : 
- Plantations bonus food delayed to Medieval Faires (from Feudalism)
- Johannesburg now give +1 production when a lux/strat/bonus is improved (one per type). +1 at Apprenticeship and +1 at Industrialization.
- Flood yields reduced by 25%.
Hello <@&1009714827721199646> 6.2c is out :
6.2c
Civilizations:
- 🔻 Brazil :
  - Minas gets -10 base ranged strength (on top of previous -10 for melee)
  - Minas gets +10 combat strength (melee and ranged) at Refining
- 🔻 Gaul: Combat bonus now only work with adjacent military gallic units
- ⬆️ Inca: Scouts get 1 charge to make a mountain tunnel (works similarly to legion with fort)
- ⬆️ Maya: Observatory gives +1 housing
- ♻️ Scythia: Combat bonus is now +3 base against wounded units, scaling to +5 when reaching medieval era.
- ♻️ Sumer :
  - Reverted previous BETA change
  - No longer starts the game with a war-cart
- ♻️ Vietnam : 
  - Reverted previous BETA change
  - Districts are now buildable on floodplains, with buildings giving +1 food
Pantheon :
- ♻️ River goddess : now grants +2 faith adjacency and +1 amenity to HS on a river (from 1 faith 1 housing 1 amenity)
- ⬆️ Sacred Path : now also includes marshes
- ⬆️ Divine spark : now gives 2 great prophet points instead of 1

City States :
- 🔻 Johannesburg : Extra prod at Apprenticeship moved to Ballistics
- ♻️ Zanzibar : Cloves are now only given after researching Banking. Amenities from both luxes reverted to 6
- ♻️ Kumasi : change to "traders to CS provide 2 culture, +1 culture and +1 gold per district in origin city" (from +2/+1 per district)

Policies & Government :
- ⬆️ Communism reworked/buffed:
  - Card slots to 2/3/1/2 (from 2/4/1/1)
  - Negative tourism modifier increased to 50% (from 40%)
- ♻️ Veterancy : 
  - Now becomes obsolete at Mobilization
  - New red card "Arms Race" replaces it : Same effect as Veterancy, but also includes aerodromes and their buildings
- ⬆️ Praetorium : +20% production on conquered cities (from +10% with a governor). Loyalty untouched.
Other:
- ♻️ Disaster 1 gets disaster 0 frequency
- ♻️ Disaster 0 gets disaster removed
**- ♻️ Spy production scales with techs/civics [need a lot of feedback on the values please]**
- ⬆️ Military Engineers can use a charge to complete 50% of a city's flood barrier production (from 20%)
- 🔻 Orszaghaz extra influence points reduced to +25% (from 50%)
- ♻️ Moksha R1 changed to "Grants 4 great prophet points if this city has a holy site. This city gets +2 faith for each city that has a holy site within 6 tiles."
- ⬆️ Moksha R2 increased to 20% culture and science from faith (from 10%)
- ♻️ Improvement pillage value reduction changed to -30% (from -40%)

Bug Fix :
**- Fixed a bug where Wu Zetian could not faith buy spies when there was an opponent spy already in the city**

Text :
- Spawn biases added to civ bonus text, overall reformatting of a lot of civ bonuses
<@&1009714827721199646> has been updated to 6.2d :
- Praetorium bug fix
-  Chinese, English, French and Korean should be up to date.
-  6.2d (changes between beta 6.2c and final 6.2.0 live version)
- Cleo-P
   - Remove river crossing movement/combat ability
   - increase combat strength in floodplains to +5 
- Chandra
   - Remove corps/army discount on tier 3 worship building
   - Each holysite building gives +1 combat strength to units built in that city
- Vietnam
   - Buildings in districts on floodplains don't give yields anymore 
- Seowon/Observatory now get +2 adjacency from geothermals
</details>

A change i missed is the -5 combat strengh to units in the ancient era

You can confirm the pillage yield decrease line 316:
https://github.com/CivilizationVIBetterBalancedGame/BetterBalancedGame/blob/main/sql/Base/base.sql

6.3 and 6.4 edits fix visual glitches

6.5:
I'm surprised this isnt in the changelog, 

<details> 
<summary>Snippet of 6.5 (change in bold)</summary>
Eurekas & Inspirations :
- ♻️ Craftman : Improve 2 tiles [3 tiles]
- ♻️ Foreign Trade : Own 1 Scout [Find new Continent]
- ♻️ Guidance Systems : Kill a unit with a Fighter [Kill a Fighter]
- ♻️ Humanism : Own 2 Amphiteater [Recruit an Artist]

Governors :
- 🔻 Moksha R1 : Great Prophet points reduced to 2. [from 4]
- 🔻 Reyna R3 : Scaling delayed to Economics. [from Banking]

Others :
- ♻️ Project Ivy : Reduced to 500 production (from 750)
**- ♻️ Military Science grant 1 Spy capacity**
- 🔻 Governor titles hut delayed to turn 25.
- 🔻 Free technology hut delayed to turn 31.

Pantheon :
- 🔻 Divine Spark : 1 Great Prophet point per turn and Engineer point need workshop. 

Policies :
- 🆕 New red card at Military Training: +10 regen for land units per turn outside of friendly territory. 

Units :
- ⬆️ Mobile SAM : Increase Strength to 125. NOTE: it will now defend against nukes.
- ⬆️ Artillery & Rocket Artillery : +5 combat strength against city center.
- 🔻 Spy : Governor and IZ missions duration increased to 6 turns.
</details>

This is line 314 of this file:
https://github.com/CivilizationVIBetterBalancedGame/BetterBalancedGame/blob/main/sql/Base/base.sql
